src/main/java/adris/altoclef/util/ItemTarget.java

```diff
@@ -27,7 +27,12 @@
     public ItemTarget(Item[] items, int targetCount) {
         _itemMatches = items;
         _targetCount = targetCount;
-        _infinite = false;
+        if (items.length > 1) {
+            //Debug.logMessage("Friendly name for " + Arrays.toString(items) + " is not provided.");
+            // Hey you reading this.
+            // If you want to help improve the user experience, go ahead and uncomment the line, so you can test it.
+            // But don't do it in the release version, because it will be annoying.
```

Comment:
I know this probably should be changed into something else, but I'm not sure whether it would make sense to have things like that in the settings.

Reply:
Having a debug mode in the settings would make it clear. We want as little code in altoclef as possible as large amounts of code that "do nothing" makes the project harder to understand later in the future. If this provides helpful information for debugging, it should have a way to be toggled. However, if it isn't really needed at all, it can be removed entirely.